l20n/migrate/__init__.py

```diff
@@ -0,0 +1,5 @@
+# coding=utf8
+
+from context import MergeContext
+from transforms import CONCAT, COPY, EXTERNAL, PLURALS, REPLACE, SOURCE
+from changesets import convert_blame_to_changesets
```

Comment:
Also, @Pike, should we maybe version the migration transforms?  As in, `from l20n.migrate.v1 import COPY`.